FRENCH/src/ch07-02-defining-modules-to-control-scope-and-privacy.md

```diff
@@ -0,0 +1,249 @@
+<!--
+## Defining Modules to Control Scope and Privacy
+-->
+
+## Définir des modules pour gérer la portée et la protection
+
+<!--
+In this section, we’ll talk about modules and other parts of the module system,
+namely *paths* that allow you to name items; the `use` keyword that brings a
+path into scope; and the `pub` keyword to make items public. We’ll also discuss
+the `as` keyword, external packages, and the glob operator. For now, let’s
+focus on modules!
+-->
+
+Dans cette section, nous allons aborder les modules et les autres outils du
+système de modules, à savoir les *chemins* qui nous permettent de nommer les
+éléments; l'utilisation du mot-clé `use` qui importe un chemin dans la portée;
+et le mot-clé `pub` que rend public les éléments. Nous verrons aussi le mot-clé
+`as`, les paquets externes, et l'opérateur glob. Pour commencer, penchons-nous
+sur les modules !
+
+<!--
+*Modules* let us organize code within a crate into groups for readability and
+easy reuse. Modules also control the *privacy* of items, which is whether an
+item can be used by outside code (*public*) or is an internal implementation
+detail and not available for outside use (*private*).
+-->
+
+Les *modules* nous permettent de regrouper le code d'une crate pour une
+meilleure lisibilité et pour la facilité de réutilisation. Les modules
+permettent aussi de gérer la *protection* des éléments, qui précise si un
+élément peut être utilisé à l'extérieur du code (*c'est public*) ou s'il est
```

Comment:
Wrong translation
```suggestion
élément peut être utilisé à l'extérieur du module (*c'est public*) ou s'il est
```
or
```suggestion
élément peut être utilisé par un code extérieur (*c'est public*) ou s'il est
```

Reply:
Yep, it's much clearer and exact in this way, thank you ! 😉